packages/proto-signing/src/directsecp256k1hdwallet.ts

```diff
@@ -243,7 +243,7 @@ export class DirectSecp256k1HdWallet implements OfflineDirectSigner {
   /** Derivation instructions */
   private readonly accounts: readonly Secp256k1Derivation[];
 
-  protected constructor(mnemonic: EnglishMnemonic, options: DirectSecp256k1HdWalletConstructorOptions) {
+  public constructor(mnemonic: EnglishMnemonic, options: DirectSecp256k1HdWalletConstructorOptions) {
```

Comment:
Looking at this constructur again, this is easy to use in a wrong way by providing a seed that does not match the mnemonic. Let's keep this protected as it was before.

If you want to make this public in a type-safe way, it is easier to derive your own wallet type:

```ts
class MyWallet extends DirectSecp256k1HdWallet {
  public constructor(yourOptions) {
    super(mnemonic, options);    
  }
}
```

This should be more powerful, type safe and allow merging this PR without further CosmJS API refinement.